### ISSUE TITLE
Use Blenders disable in viewport/renderer options for ignoring STK objects

### ISSUE THREAD

**Issue by JulianSchoenbaechler** [JulianSchoenbaechler/stk-blender#10]
Everything marked as disabled in the Outliner or disabled collections should be ignored from the exporters as well. This removes the necessity of the `Ignore` object property.

**Comment by RQWorldblender** [comment]
My latest commit starts on this part: you can use hide_render to prevent certain objects from being exported. It has the same effect as if the STK type of an object is "ignore", but using native Blender functionality. 02e60df

**Comment by JulianSchoenbaechler** [comment]
Thanks, I noticed

By now, the add-ons core has changed a lot (check the commits on the heavy work-in-progress [`workflow-improvements`](/JulianSchoenbaechler/stk-blender/tree/workflow-improvements) branch.

After some thinking, I've decided to keep the `Ignore` option additionally to the disable option in Blender itself. Sometimes it is really useful to have an object visible at all times in the viewport, but prevent it from getting exported.
https://github.com/JulianSchoenbaechler/stk-blender/blob/09981fa5c7a0e083373c8db189650bed75a3f52c/io_antarctica_scene/stkdata/stk_track_object_properties.xml#L8-L9
